app/Http/Resources/V2/Dashboard/ActiveProjectsTableResource.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace App\Http\Resources\V2\Dashboard;
+
+use Illuminate\Http\Resources\Json\JsonResource;
+
+class ActiveProjectsTableResource extends JsonResource
+{
+    /**
+     * Transform the resource into an array.
+     *
+     * @param  \Illuminate\Http\Request  $request
+     * @return array
+     */
+    public function toArray($request)
+    {
+        return $this->resource;
```

Comment:
These resource classes are really only useful when an actual conversion needs to take place (like from properties on an Model to a JSON-encodable array). A Resource class that doesn't do anything other than return the value passed into it can be removed, and the controller method can simply use `response()->json([...])` instead of returning the custom resource class instance.

Reply:
Removed unnecesary class.